src/module/cp/oierdb.py

```
@@ -0,0 +1,370 @@
+from src.core.bot.message import RobotMessage
+from src.data.data_oierdb import oierdb_instance
+
+@command(tokens=["oier", "OI选手", "信息学奥赛"])
+def query_oier(message: RobotMessage):
+    """查询OI选手信息"""
+    try:
+        # 移除@标签并解析命令
+        content = re.sub(r'<@!\d+>', '', message.content).strip().split()
+        
+        if len(content) < 1:
+            return message.reply("请输入命令，如: oier 张三", modal_words=False)
```

Ruff (0.13.3)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)